README.md

```diff
@@ -168,6 +168,11 @@ When using sidecar mode the OpenTelemetry collector container will have the envi
 
 The operator can inject and configure OpenTelemetry auto-instrumentation libraries. Currently Java, NodeJS and Python are supported.
 
+Each auto-instrumentation supports different exporters. Make sure your endpoint is configured properly:
+* Java - by default `OTLP gRPC exporter` is used (default port 4317). To change configuration please see [OT Java Auto-Instrumentation](https://github.com/open-telemetry/opentelemetry-java/blob/main/sdk-extensions/autoconfigure/README.md#exporters)
```

Comment:
I like the list, however, what are the implications for the user? We should document what value of `spec.exporter.endpoint` should be set for each language. 

Then if the exporter endpoint differs per language how can the CR be configured for multiple languages?

Reply:
I've added little bit about `Instrumentation` resource and some information about each supported auto-instrumentation. I think it will make easier for end user to run it.